m2cgen/interpreters/java/code_generator.py

```diff
@@ -15,7 +15,9 @@ def add_class_def(self, class_name, modifier="public"):
 
     def add_method_def(self, name, args, return_type, modifier="public"):
         method_def = modifier + " static " + return_type + " " + name + "("
-        method_def += ",".join([t + " " + n for t, n in args])
+        method_def += ",".join([
```

Comment:
It's a bit confusing that we still need to figure out the var type for arguments in this method, but `return_type` already contains a type itself.